### PR TITLE
Fixes #13 - Schema of home page in React - Azin

### LINKED ISSUE #13: Schema of Home page in React

### PR DESCRIPTION
**Pull request by AzinYad**
## Description

I have deleted all extra pages except for home page for this step as you suggested , I also Used React Icon Package instead of inserting several png files.
Css for this page is also responsive , for smaller device I will add Media Query later if it was needed

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

**Comment by bertilnilsson** [comment]
Sorry, I'm might be misreading your description. This PR still includes all the new pages and components as well as the various png and svg files? Is that a mistake?

**Comment by AzinYad** [comment]
> Sorry, I'm might be misreading your description. This PR still includes all the new pages and components as well as the various png and svg files? Is that a mistake?

you are right I'm so sorry I thought as I deleted the file they will be deleted on the branch It seems I have to deleted them in terminal as well ,Im on it

File: client/src/App.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import { Route, Routes } from "react-router-dom";
 
-import About from "./pages/About";
+// import About from "./pages/About";
```
Comment (bertilnilsson) [inline client/src/App.js]: Why not just remove rather than comment out since the page is being removed?

File: client/src/App.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+
 import Home from "./pages/Home";
 
 const App = () => (
 	<Routes>
-		<Route path="/" element={<Home />} />
-		<Route path="/about/this/site" element={<About />} />
+		<Route path="/home" element={<Home />} />
```
Comment (bertilnilsson) [inline client/src/App.js]: Why change the route from / to /home?

File: package.json [hunk 3]
```diff
@@ -72,11 +72,15 @@
 	},
 	"dependencies": {
 		"@babel/runtime": "^7.20.13",
+		"@fortawesome/fontawesome-svg-core": "^6.3.0",
+		"@fortawesome/free-solid-svg-icons": "^6.3.0",
+		"@fortawesome/react-fontawesome": "^0.2.0",
```
Comment (bertilnilsson) [inline package.json]: Are these packages being used?

**Comment by bertilnilsson** [comment]
I posted some review comments above. 

Also will you close your other PR #11?

**Comment by AzinYad** [comment]
> I posted some review comments above.
> Sorry I couldn't find any review on the code ,except for suggesting to take this issue back to draft if it is still in progress,as a schema of home page I think its done so I moved it to that section ,not sure if it was right !
  can you guide me about where can I check your review 
  
> Also will you close your other PR #11?
I closed PR#11

**Comment by bertilnilsson** [comment]
The review is visible above in this thread or on the lines of code on Files changed tab.

**Comment by AzinYad** [comment]
> The review is visible above in this thread or on the lines of code on Files changed tab.

I can see them now ,thanks

**Comment by bertilnilsson** [comment]
Sorry, review had not been posted - it was still pending as a draft.

File: package.json (hunk 3 above)
Comment (AzinYad) [reply package.json]: I have Uninstalled them

**Review by bertilnilsson — APPROVED** [review]

**Review by RobCso — APPROVED** [review]
Can't see issues, I approve this PR

**Pull request merged**